apps/src/lib/util/audioApiDropletConfig.js

```diff
@@ -32,6 +32,19 @@ const dropletConfig = {
       0: () => getAssetDropdown('audio')
     },
     assetTooltip: {0: chooseAsset.bind(null, 'audio')}
+  },
+  playSpeech: {
+    func: 'playSpeech',
+    parent: executors,
+    paramButtons: {minArgs: 2, maxArgs: 3},
+    paletteParams: ['text', 'gender', 'language'],
+    params: ['"Hello World!"', '"female"', '"en-us"'],
+    dropdown: {
+      1: ['"female"', '"male"'],
+      2: ['"en-us"']
```

Comment:
I'd suggest we use the common name of the language, rather than the language code. Like what we do for the language dropdown in the footer